src/components/home_complaints/index.js

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+
+import { Loading } from "formula_one";
+
+import locals from "./index.css";
+import styles from "core/index.css";
+
+import { complaintsUrl } from "../../urls";
+import { getComplaints } from "../../actions/complaints";
+import { addComplaint } from "../../actions/add_complaint";
+import moment from "moment";
+import { urlStaticBase } from "../../urls";
+
+class HomeComplaints extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      complaints: {},
+      max_length: 6,
+      loading: true,
+    };
+  }
+
+  componentDidMount() {
+    this.props.getComplaints(
+      `${complaintsUrl(this.props.activeHostel)}?me=True`,
+      this.complaintsSuccessCallBack,
+      this.complaintsErrCallBack
+    );
+  }
+
+  complaintsSuccessCallBack = (res) => {
+    this.setState({
+      loading: false,
+      complaints: res.data,
+    });
+  };
+
+  complaintsErrCallBack = (err) => {
+    this.setState({
+      loading: false,
+    });
+  };
+
+  render() {
+    const { complaints, constants } = this.props;
+    const { loading } = this.state;
+
+    return (
+      <div className={`${styles["my-12"]} ${styles["w-full"]}`}>
+        <div
+          className={`${styles["flex"]} ${styles["flex-row"]} ${locals["gap-16"]}`}
+        >
+          <img
+            src={`${urlStaticBase()}/complaints.png`}
```

Comment:
-> Check if Tailwind has a icon set and if present use that instead of image.

Reply:
No, tailwind doesn't have any icon set. We can use heroicons, font-awesome etc. for icons.